plasma_framework/docs/deploying/deploying.md

```diff
@@ -12,6 +12,11 @@ Truffle migration scripts run in the order of the numbered prefix of the filenam
 - 2000: Future extensions, new exit games, etc.
 
 ## ENV VAR settings
+- DEPLOYER_PRIVATEKEY: Private key of the deployer. Required when deploying with `--network remote`.
```

Comment:
`PRIVATEKEY` or `PRIVATE_KEY`?

Reply:
this is how it is used for now 🤷‍♂️. I agree I don't like the var name, but I don't want to introduce change now.
Or actually if DevOps deployment flow is not too much impacted we can change that.... cc. @jbunce @arthurk 

https://github.com/omgnetwork/plasma-contracts/blob/804a4f054bab7d46644e57ef0069b775fc645264/plasma_framework/truffle-config.js#L30